test/test_pipe_baseline.py

```diff
@@ -1,21 +1,53 @@
 import unittest
+import resonance.pipe as pipe
+import resonance.si as si
+import resonance.db as db
 import resonance.run as run
+import numpy as np
 
 
 class TestPipeBaseline(unittest.TestCase):
-    def check(self, si, blocks, expected):
-        code = '''
-from resonance import *
-in = input(1)
-out = pipe.baseline(in)
-createOutput(out, 'out')
-'''
+    def check(self, si, blocks, expected, beginOffset, endOffset):
+#        code = '''
+# from resonance import *
+# inputData = input(0)
+# out = pipe.baseline(inputData, beginOffset, endOffset)
```

Comment:
This is a string, you should put argument in a string directly
use .format for now